Documentation/CACHE-HOWTO.md

```diff
@@ -134,7 +133,7 @@ Add `executionControl/runAlways` section
 ```xml
 <?xml version="1.0" encoding="UTF-8" ?>
 <cache xmlns="org:apache:maven:cache:config:v1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
The namespace is wrong. If you want to use a URN, then you must start with `urn:`

Reply:
I think it may be better to review the changes rather than the original submission. 
If you have anything that you think should be changed, it may be easier to add a comment to https://issues.apache.org/jira/browse/MNG-7129, I'd be happy to fix those.

Reply:
Agreed.

Reply:
Fwiw, I think the namespace should be changed to be inline with other maven namespaces and not using urns.

Reply:
Absolutely, that was my first thought too.